.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,54 @@
+name: E2E test
```

Comment:
This should be renamed to avoid confusion with other planned e2e worklows. Maybe something like `e2e-pr` (filename) and `E2E - PR` (name)

Reply:
This can be run also manually, so I would rather omit the PR part.